roles/zabbix_server/tasks/main.yml

```diff
@@ -31,13 +31,18 @@
 - name: Install the correct repository
   ansible.builtin.include_tasks: "{{ ansible_os_family }}.yml"
 
-- name: Install zabbix-server package
+- name: Install zabbix-server packages
   ansible.builtin.package:
-    name: "{{ zabbix_server_package | default(_zabbix_server_package) }}"
+    name: "{{ item.name }}"
     state: "{{ zabbix_server_package_state }}"
     update_cache: true
     disablerepo: "{{ zabbix_server_disable_repo | default(_zabbix_server_disable_repo | default(omit)) }}"
     install_recommends: "{{ zabbix_server_install_recommends | default(_zabbix_server_install_recommends | default(omit)) }}"
+  when: item.required | default(true)
+  loop:
+    - name: "{{ zabbix_server_package | default(_zabbix_server_package) }}"
+    - required: "{{ zabbix_server_version is version('6.0', '>=') }}"
```

Comment:
OK maybe you can help me understand what you're doing here.  What is the required parameter doing?  I don't actually see it being used anywhere?

Reply:
It's getting used in the when condition above. (Though all this code has since been removed with the 5.0 removal)